### ISSUE TITLE
Website mobile usability 

### ISSUE THREAD

**Issue by IlanLuci** [OpenLitterMap/openlittermap-web#48]
Love this project! Here are some ideas for improvements to the website. 
1. On mobile the footer horizontal line divider cuts into the single donation crowdfunding link. 
2. I think the menu should close when you load a new page. Right now when you load to a new page the menu stays open and you have to click X to close it. 
3. The about page needs a decent amount of work on mobile. some of the images sizes are cutting into the edge of the page and we have a horizontal page scroll.

**Comment by xlcrr** [comment]
Thanks! These are all on the todo list, feel welcome to submit a PR! We would love if you could help us.

**Comment by xlcrr** [comment]
About page also needs to lazy-load images, and we could probably reduce the size of them too

**Comment by IlanLuci** [comment]
Yes! I am doing a little work on that right now and I am trying to figure out why there is white space on the right of the screen creating a horizontal scroll. I think something has margin-right because the widths of everything seems to be fine.

**Comment by xlcrr** [comment]
What page has maring-right? We are currently using bulma columns everywhere but we need to stop using these as they come with padding and margins which is definitely messing up on mobile devices

**Comment by IlanLuci** [comment]
I'm pretty sure the about page does. On mobile there is white space to the right and there are no elements In the white space.
